README.md

```diff
@@ -8,3 +8,150 @@ A platform for MLH Fellows to share their stories and experiences.
+- Easy search functionality
+
+## Challenges we ran into
+
+  1. All of us are new to Docusaurus, took us sometime to explore it.
+  2. Integrating backend to frontend is always time consuming, so that's one challenge.
+  3. Time management was really challenging since it's only the beginning of the fellowship
+  
+## Accomplishments that we're proud of
+
+  1. Fellow Stories is one of it's kind, exclusively for the MLH Fellows, by the MLH Fellows
+  2. Fellows now have a dedicated platform for all their stories and experiences
+  3. This would bring the community closer, and help it expand even more
+  4. This would help in setting up a dedicated readership for the fellowship
+
+## What we learned
+  1. It was really interesting for us to explore Docusaurus. We were all new to this, thus took a while to explore and understand how things work within Docusaurus before actually starting to code. 
+  2. We all went through rigorous meetings and discussions in this phase, which helped us in understanding more about collaboration to an open source project.
+  3. We learned and implemented open source best practices like- making use of GitHub project board, setting up branch protection rules, adding Issue/PR templates, and following a consistent style for writing commit messages etc.
+  4. We made use of some GitHub bots like - Welcome Bot, Stale Bot, Gitpod etc so as to speed collaboration and project management.
+
+
+## What's next for Fellow Stories
+
+- [ ] Create a MLH Fellowship Email Newsletter to update people on new posts
+- [ ] Integrate with GitHub for easy access to recent updates in posts
+- [ ] Option to customize and manage favorite content on dashboard
+- [ ] Add tags for filter and search
+
+## How to Get Started?
+
+[![Requirements Up To Date](https://img.shields.io/badge/requirements-up%20to%20date-brightgreen)](https://github.com/PragatiVerma18/Fantastic-Falcons-1.0/blob/webapp/requirements.txt)
+[![Python](https://img.shields.io/badge/python-v3.7-blue)](https://www.python.org/)
+
+### GitHub Repository Structure
+
+| S.No. | Branch Name                                                                   | Purpose                            |
+| ----- | ----------------------------------------------------------------------------- | ---------------------------------- |
+| 1.    | [main](https://github.com/MLH-Fellowship/FellowStories/tree/main) | contains the frontend related code             |
+| 2.    | [backend](https://github.com/MLH-Fellowship/FellowStories/tree/backend)       | contains all backend related code |
+| 3.    | [documentation](https://github.com/MLH-Fellowship/FellowStories/tree/documentation) | contains all documentation |
+
+### Setup
+
+- Fork and Clone the repo using
+
+```
+$ git clone https://github.com/MLH-Fellowship/FellowStories.git
+$ cd FellowStories
+```
+
+- Install all npm dependencies using
+```
+npm i
+```
+
+- Run the server using
+```
+npm start
+```
```

Comment:
Our frontend project is using yarn, so we can change the commands to yarn, or if backend project is using npm then have separate sections for frontend and backend.
- `npm i` changes to `yarn`.
- Change `npm start` changes to `yarn start`

Reply:
Done 🚀 Let me know if anything else needs to be changed 😄